apps/ledger-live-mobile/src/families/cosmos/RedelegationFlow/01-SelectValidator.js

```diff
@@ -101,10 +101,7 @@ function RedelegationSelectValidator({ navigation, route }: Props) {
       validators
         .reduce(
           (data, validator) => {
-            if (
-              validator.validator.validatorAddress ===
-              transaction?.sourceValidator
-            )
+            if (validator.validatorAddress === transaction?.sourceValidator)
```

Comment:
Similar question, is this fixing a regression for Cosmos on LLM?

Reply:
This was weird. We encountered issues with Redelegating from LLM and we noticed that at some point a regression must have been introduced because the latest `develop` shows the line as we re-introduced it in this PR. Not sure exactly how the regression was introduced: https://github.com/LedgerHQ/ledger-live/blob/b81f8adf4ae968a7ccdca887032b14d23d4522aa/apps/ledger-live-mobile/src/families/cosmos/RedelegationFlow/01-SelectValidator.js#L103